### PR TITLE
docs: add missing instruction in HTTP section

### PR DESCRIPTION
**Pull request by FrederikSchlemmer**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

The current behavior is that the steps in the "HTTP" section of the Tour of Heroes cause confusion for the user. A step for creating the file "in-memory-data.service" is missing.

The problem has been reported by others in Stack Overflow:
[https://stackoverflow.com/questions/39877282/cannot-find-module-in-memory-data-service-in-tour-of-heroes-for-angular2]()

Issue Number: #25693

## What is the new behavior?
The new behavior is that the step to create the file "in-memory-data.service" is explained correctly.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by brandonroberts** [comment]
Shouldn't be a `class` that's generated and not a service?

File: aio/content/tutorial/toh-pt6.md [hunk 1]
```diff
@@ -77,8 +77,13 @@ _after importing the `HttpClient`_,
 The `forRoot()` configuration method takes an `InMemoryDataService` class
 that primes the in-memory database.
 
-The _Tour of Heroes_ sample creates such a class 
-`src/app/in-memory-data.service.ts` which has the following content:
+Now the class `src/app/in-memory-data.service.ts` is generated by the following command:
```
Comment (brandonroberts) [inline aio/content/tutorial/toh-pt6.md]: "Now the" -> "The"

File: aio/content/tutorial/toh-pt6.md [hunk 2]
```diff
@@ -77,8 +77,13 @@ _after importing the `HttpClient`_,
...
+
+<code-example language="sh" class="code-shell">
+  ng generate service InMemoryData --module=app
```
Comment (brandonroberts) [inline aio/content/tutorial/toh-pt6.md]: Remove `--module=app` as this is no longer supported in the CLI.

File: aio/content/tutorial/toh-pt6.md [hunk 3]
```diff
@@ -77,8 +77,13 @@ _after importing the `HttpClient`_,
...
+</code-example>
+
+This class has the following content:
 
 <code-example path="toh-pt6/src/app/in-memory-data.service.ts" region="init" title="src/app/in-memory-data.service.ts" linenums="false"></code-example>
```
Comment (brandonroberts) [inline aio/content/tutorial/toh-pt6.md]: The code snippet here needs to be updated also with the default syntax generated for a service.

```
@Injectable({
  providedIn: 'root'
})
export ...
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: aio/content/tutorial/toh-pt6.md (hunk 3 above)
Comment (FrederikSchlemmer) [reply aio/content/tutorial/toh-pt6.md]: Also updated the code snippet!

**Comment by FrederikSchlemmer** [comment]
@brandonroberts When updating the code snippet I noticed that the example on stackblitz would also need to be updated.
Could you briefly explain to me how to update from stackblitz examples?

Live Example:
https://stackblitz.com/angular/xkxkdjbqaol

**Comment by brandonroberts** [comment]
@FrederikSchlemmer the stackblitz examples are generated at build time, so as long as your change is included in the doc regions it will be added in the example.

File: aio/content/examples/toh-pt6/src/app/in-memory-data.service.ts [hunk 4]
```diff
@@ -1,5 +1,10 @@
 // #docregion , init
 import { InMemoryDbService } from 'angular-in-memory-web-api';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
```
Comment (brandonroberts) [inline aio/content/examples/toh-pt6/src/app/in-memory-data.service.ts]: Move the decorator right above the class.

```
@Injectable({
  providedIn: 'root',
})
export class ...
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Comment by FrederikSchlemmer** [comment]
@brandonroberts Good to hear that the stackblitz example is updated automatically! I have applied the changes.

**Review by brandonroberts — APPROVED** [review]

**Comment by FrederikSchlemmer** [comment]
@brandonroberts I made a rebase on this pull request, which requires a new approval.

I hope that the PR will be merged soon...

**Comment by FrederikSchlemmer** [comment]
@brandonroberts What do you think about this PR?

**Comment by brandonroberts** [comment]
@FrederikSchlemmer PR is still good. Will you rebase this branch again? Still waiting on a second review/approval.

**Comment by FrederikSchlemmer** [comment]
@brandonroberts I did the rebase. 
Ready for the next review @IgorMinar @jenniferfell.

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**